.github/workflows/test_and_build.yml

```diff
@@ -46,6 +46,8 @@ jobs:
         pipenv sync
         pipenv run pip install matplotlib
         pipenv run pip install pytest-asyncio
+        pipenv run pip install shapely
+        pipenv run pip install geopandas
```

Comment:
Should not be necessary if applying the change in Pipfile (moving deps from dev-packages to packages).
```suggestion
```